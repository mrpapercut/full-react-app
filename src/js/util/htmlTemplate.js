const htmlTemplate = ({title, body, scriptTags, preloadedState}) => `<!doctype html>
<html>
<head>
    <meta charset="utf8"></meta>
    <title>${title}</title>
    <base href="/">

    <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
    <div id="appwrapper">${body}</div>
    <script>${preloadedState}</script>
    ${scriptTags}
</body>
</html>
`;

export default htmlTemplate;
