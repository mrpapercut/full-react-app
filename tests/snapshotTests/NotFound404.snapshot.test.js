import {createElement} from 'react';
import renderer from 'react-test-renderer';

import {wrapElementWithRouter} from '../util/snapshot-test-util';

import NotFound404 from '~/components/NotFound404';

test('NotFound404', () => {
    const component = renderer.create(wrapElementWithRouter(createElement(NotFound404)));
    expect(component.toJSON()).toMatchSnapshot();
});
