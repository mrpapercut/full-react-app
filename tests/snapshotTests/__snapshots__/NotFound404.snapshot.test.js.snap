// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound404 1`] = `
<div
  className="notfound404"
>
  <h2
    className="pagetitle"
  >
    404 - Page not found
  </h2>
</div>
`;
